src/main/java/com/spanish_inquisition/battleship/client/MainMenuController.java

```diff
@@ -94,5 +101,18 @@ private void runTheGame() {
         this.playersLabel.setText("Set Up your ships");
         this.game = new Game();
         this.game.buildPlayersBoard(new BoardController(new GameBoard(this.playersGridPane)));
+        this.fleetSetupButton.setVisible(true);
+    }
+
+    @FXML
+    public void onFleetSetupButtonClicked(){
+        this.game.placePlayersShips();
+        this.sendToServerButton.setVisible(true);
```

Comment:
Why always "this"? Method without parameters and "this", "this", "this"? Defensive programming!